spk/readarr/Makefile

```
@@ -0,0 +1,49 @@
+SPK_NAME = readarr
+SPK_VERS = $(shell date +%Y%m%d)
```

```suggestion
SPK_VERS = 0.1.0.1225
```

IMHO Using a shell date is a last resort if no version number is available.

---

This should be reverted to `$(shell date +%Y%m%d)` as all packages that use internal package updates should not define a specific version number.

---

This change suggestion was not committed so nothing to revert at the moment. Should some other best practice be followed rather than resolving the conversation?